docs/DevGuide/GpuSupport.md

```diff
@@ -17,6 +17,11 @@ library.
 
 ## Build configuration
 
+You can use either NVIDIA's NVCC compiler (which comes with the CUDA toolkit) or
```

Comment:
Not a change request yet, but I need to confirm these instructions are sufficient for me to reproduce this build on a machine I have access to so when this CI build fails on my PR I can attempt to fix the problem